util/reggen/uvm_reg.sv.tpl

```diff
@@ -83,26 +96,28 @@ package ${block.name}_ral_pkg;
         .is_rand(1),
         .individually_accessible(1));
       ${f.name}.set_original_access("${field_access}");
-  % if r.hwext:
-      set_is_ext_reg(1);
-  % endif
-  % if len(r.fields) == 1:
-      add_hdl_path_slice("u_${reg_name}.q", ${f.lsb}, ${field_size});
+  % if f.hwaccess == HwAccess.NONE and f.swrdaccess == SwRdAccess.RD and f.swwraccess == SwWrAccess.NONE:
+      // constant reg
+      add_hdl_path_slice("${hier_path}u_reg.${reg_field_name}_qs", ${f.lsb}, ${field_size});
   % else:
-      add_hdl_path_slice("u_${reg_name}_${f.name}.q", ${f.lsb}, ${field_size});
+      add_hdl_path_slice("${hier_path}u_reg.u_${reg_field_name}.q${"s" if r.hwext else ""}", ${f.lsb}, ${field_size});
```

Comment:
@eunchan 
could you confirm if these paths are right? if it's not hwext, the reg storage is at `u_${reg_field_name}.q`. If it's hwext, the reg value is connected to`u_${reg_field_name}.qs`, which we may use for backdoor read.

Reply:
For external, yes.

Reply:
Thanks for confirming.